client/src/services/api-abstractions.ts

```diff
@@ -10,27 +9,27 @@ interface ApiAbstractions {
 }
 
 export interface FindApi<TFindResult> extends ApiAbstractions {
-	findAsync: (id:string) => Promise<TFindResult|ApiError|null>;
+	findAsync: (id:string, accessToken:string) => Promise<TFindResult|ApiError|null>;

```

Comment:
Would it be better to pass the accessToken into the specific API class constructors instead of per-method?

Or, just have the API class constructors retrieve the token from the session itself?